infrastructure.md

```diff
@@ -37,15 +39,22 @@ Please contact [Adam Perry](https://github.com/dikaiosune) ("dikaiosune" on IRC)
 
 [DXR](https://dxr.mozilla.org/rust/source/) is a cross-referenced source index for Rust, allowing the Rust source tree to be navigated and searched with ease. It is generated by [rust-dxr](https://github.com/nrc/rust-dxr)
 
-## Others
+## perf
+
+[perf](https://github.com/rust-lang-nursery/rustc-perf) offers information about the performance of `rustc` over time.
+
+It is split into a data [collector](https://github.com/rust-lang-nursery/rustc-perf/tree/master/collector) and a [web frontend](https://github.com/rust-lang-nursery/rustc-perf/tree/master/site). The raw performance data is available [here](https://github.com/rust-lang-nursery/rustc-timing) and can be browsed on the [perf website](https://perf.rust-lang.org).
+
+## rust release explorer
+
+[rust release explorer](https://github.com/ashleygwilliams/rust-release-explorer/) is a way to explore rust release features based on milestone labels. The website can be found [here](https://ashleygwilliams.github.io/rust-release-explorer).
```

Comment:
cc @ashleygwilliams -- should we link to this? It looks somewhat unmaintained (though the release team might be interested in taking it over if you're willing).